apps/src/music/blockly/toolbox/toolboxBlocks.ts

```diff
@@ -11,8 +11,12 @@ import {
   TriggerStart,
 } from '../constants';
 
-const toolboxBlocks: {[blockType in BlockTypes | string]: BlockInfo} = {
+// Blockly's BlockInfo type with an additional 'levelbuilderText' field used for levelbuilder-facing English block text.
+type BlockInfoWithText = BlockInfo & {levelbuilderText: string};
```

Comment:
Note: Depending on which branch merges first (this one or https://github.com/code-dot-org/code-dot-org/pull/61025), we'll need to add `levelbuilderText` to the new effect blocks in the unmerged branch. Otherwise we'll start getting a `Property 'levelbuilderText' is missing` error.

Reply:
Level data JSON editing was just merged this morning, so we should have a way to author that in case we don't have a GUI control for it

Reply:
Nice! Less urgent then, especially since I think it's the same stakeholder group that wanted both the JSON editor and the AI config option. :)

Reply:
> Note: Depending on which branch merges first (this one or #61025), we'll need to add `levelbuilderText` to the new effect blocks in the unmerged branch. Otherwise we'll start getting a `Property 'levelbuilderText' is missing` error.

Prepped in https://github.com/code-dot-org/code-dot-org/pull/61100.